### ISSUE TITLE
'Microsoft.Web/serverfarms/firstPartyApps/settings' doesn't have types available

### ISSUE THREAD

**Issue by 49502291** [Azure/bicep-types-az#1843]
**Bicep version**
 0.4.1008

**Describe the bug**
Resource type 'Microsoft.Web/serverfarms/firstPartyApps/settings' doesn't have types available

**To Reproduce**
I need define Geneva configs as below, but get type unavailable. Looks like bicep has no support to Geneva configs

```bicep
resource antMDSConfigXML 'Microsoft.Web/serverfarms/firstPartyApps/settings@2019-08-01' = {
  name: '${aspName}/AntMDS/MdsConfigXml'
  location: location
  properties: {
    firstPartyId: 'AntMDS'
    settingName: 'MdsConfigXml'
    settingValue: genevaConfig.configXml
  }
}
```

**Comment by shcherbin** [comment]
Hello @alex-frankel , @naveedaz, @seligj95, any news on this issue?
If not, is there a known workaround to add these settings in a bicep template without the resource?

**Comment by anthony-c-martin** [comment]
> Hello @alex-frankel , @naveedaz, @seligj95, any news on this issue? If not, is there a known workaround to add these settings in a bicep template without the resource?

Yes - you can just suppress the warning or ignore it. Bicep will not block the resource from being deployed, the warning is just indicating we're unable to validate the properties for correctness.

**Comment by dciborow** [comment]
> > Hello [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/u/11078275?s=64&amp;u=6472c04906ab3bb749dbd69466d7a0e79249fe12&amp;v=4">@alex-frankel](https://github.com/alex-frankel) , [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/naveedaz?size=32">@naveedaz](https://github.com/naveedaz), [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/seligj95?size=32">@seligj95](https://github.com/seligj95), any news on this issue? If not, is there a known workaround to add these settings in a bicep template without the resource?
> 
> Yes - you can just suppress the warning or ignore it. Bicep will not block the resource from being deployed, the warning is just indicating we're unable to validate the properties for correctness.

Is there a way to just suppress this warning for this resource without doing it for the whole project?
